README.adoc

```diff
@@ -6,13 +6,48 @@ ____
 
 == Create a VM
 
-. launch my interactive menu
+. launch my interactive menu using the *_create_vm.sh_* script (located in the "scripts" folder)
```

Comment:
the informations are repeat two times.
1. once in this sentence.
2. a second time in the command below.
should we shoot one of the both ?

Reply:
Definitely, the "image" of the menu could directly go with 1.

Reply:
What is `the "image" of the menu` ?

Reply:
Just the display of the menu, the different options